src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/ContinuousTabularDistribution/Builder.hpp

```diff
@@ -0,0 +1,28 @@
+template< typename ParentBuilder >
+class Builder: 
+  public Tabulated< Data >::Builder< Builder< ParentBuilder > >,
```

Comment:
I can't do that because then I can't cast back to the derived builder class like I do [here](https://github.com/njoy/ACEtk/blob/f6d8c6f050e51fe4f39b5191f4eaa54c9a34f29f/src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/Tabulated/Builder/src/boundaries.hpp#L8)